app/src/main/java/com/bogdan/codeforceswatcher/util/StringExtensions.kt

```diff
@@ -9,3 +9,11 @@ fun String.convertFromHtml(): String {
 
     return HtmlCompat.fromHtml(normalizedText, HtmlCompat.FROM_HTML_MODE_LEGACY).trim().toString()
 }
+fun String.splitStringInHalf(): Pair<String, String> {
```

Comment:
Interesting workaround, but can be buggy. Are you sure we absolutely need it? Anyway this can be moved to common.

Reply:
without it big contest names doesn't fit into screen

Reply:
@MatyashDen I suspected it :) Probably you can also check the size of string and break lines only if needed. Or try to find the way to limit the size of popup with rating change and then break lines automatically.

P.S. It's already good enough, but if you want a challenge, here is one 👍